test/integration/physics_system.cc

```diff
@@ -1333,3 +1334,46 @@ TEST_F(PhysicsSystemFixture, NestedModelIndividualCanonicalLinks)
   EXPECT_NE(postUpModelPoses.end(), modelIt);
   EXPECT_NEAR(zExpected, modelIt->second.Z(), 1e-2);
 }
+
+/////////////////////////////////////////////////
+TEST_F(PhysicsSystemFixture, PhysicsOptions)
```

Comment:
Would it be worth adding another test that checks the default values for `collisionDetector` (`fcl`) and `solverType` (`dantzig`)? That way, we can confirm that both default and user-defined input behave as expected.

Reply:
The components were not being populated by default, only when the user set custom options. I think it's useful for users to know the default values, if available, so I added that in 144adfbfb132003179fedc527e590bf67d4e6617